### PR TITLE
Change hue tests

### PR DESCRIPTION
**Pull request by atabak-alishiri**
The function is now getting an Image object and after changing hue returning an Image object. test units also changed accordingly.

### CODE REVIEW

**Review by SoloSynth1 — COMMENTED** [review]
The function looks fine! Would you mind to change the testing function names so that the expected behaviours are stated in the name? Thanks!

File: tests/test_change_hue.py [hunk 1]
```diff
@@ -1,6 +1,29 @@
 from visumorph.change_hue import change_hue
 import pytest
+from visumorph import Image as VImage
+from visumorph import load_image
+import numpy as np
+from PIL import Image as PImage
```
Comment (SoloSynth1) [inline tests/test_change_hue.py]:
```suggestion
```

Comment (SoloSynth1) [reply tests/test_change_hue.py]: The tests are not using PIL (for now)

**Comment by atabak-alishiri** [comment]
> The function looks fine! Would you mind to change the testing function names so that the expected behaviours are stated in the name? Thanks!

do you mean like changing `test_different_color()` to `test_change_hue_uses_default_values_correctly()`

**Comment by SoloSynth1** [comment]
> > The function looks fine! Would you mind to change the testing function names so that the expected behaviours are stated in the name? Thanks!
> 
> do you mean like changing `test_different_color()` to `test_change_hue_uses_default_values_correctly()`

Yes, and also something like `test_string_in_delta_hue_should_return_error()`

**Comment by atabak-alishiri** [comment]
> > > The function looks fine! Would you mind to change the testing function names so that the expected behaviours are stated in the name? Thanks!
> > 
> > 
> > do you mean like changing `test_different_color()` to `test_change_hue_uses_default_values_correctly()`
> 
> Yes, and also something like `test_string_in_delta_hue_should_return_error()`

I should be okay now

**Review by SoloSynth1 — APPROVED** [review]
LGTM, the docstrings is modified and is now missing examples. I will add it back later on.

**Comment by SoloSynth1** [comment]
Merging to `main`.

**Pull request merged**